data/migrations/20210713225521_updated-table.js

```diff
@@ -16,6 +16,7 @@ exports.up = function (knex) {
       incidents.string('status').defaultTo('pending').notNullable();
       incidents.float('confidence');
       incidents.string('tags').notNullable();
+      incidents.string('src');
```

Comment:
Pending tests with legacy data, this will require a 10,000 char limit. I'll get back to you on that.